.NET/Microsoft.Recognizers.Text.DateTime/Utilities/AgoLaterUtil.cs

```diff
@@ -28,6 +28,19 @@ public static class AgoLaterUtil
 
         public static List<Token> ExtractorDurationWithBeforeAndAfter(string text, ExtractResult er, List<Token> ret,
                                                                       IDateTimeUtilityConfiguration utilityConfiguration)
+        {
+            if (utilityConfiguration.CheckBothBeforeAfter)
+            {
+                return ExtractorDurationWithBeforeAndAfterInBeforeAndAfterString(text, er, ret, utilityConfiguration);

```

Comment:
Can there also be some simplification here? I didn't get why two methods need to exist.
Couldn't it be the same, controlled by a parameter?

Reply:
@tellarin, here we have duplicated the method for readability, since the original one is shorter and easier to review. But there is no need for it, we will merge the two.

Reply:
merged